agent/grpc/handler.go

```diff
@@ -33,11 +33,13 @@ func NewHandler(logger Logger, addr net.Addr, register func(server *grpc.Server)
 		middleware.WithUnaryServerChain(
 			// Add middlware interceptors to recover in case of panics.
 			recovery.UnaryServerInterceptor(recoveryOpts...),
+			//otelgrpc.UnaryServerInterceptor(),
```

Comment:
We can't use gRPC yet because it needs a much higher version of the gRPC libraries which pushes the envoy go-control-plane higher than Consul can support right now